simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/component/ocppcommctrlr/NetworkConfigurationPriorityVariableAccessorTest.java

```diff
@@ -77,10 +76,7 @@ void getVariableSetters() {
 
     @Test
     void getVariableValidatorsValidValue() {
-        Map<Integer, ConnectionData> networkConnectionProfiles = new HashMap<>() {{
-            put(1, new ConnectionData());
-            put(2, new ConnectionData());
-        }};
+        Map<Integer, ConnectionData> networkConnectionProfiles = Map.of(1, new ConnectionData(), 2, new ConnectionData());
```

Comment:
lovely, I forgot you are on java 11 already :-) 
(if you like you can replace ImmutableMap.of with Map.of, maybe we can cut out guava soon :-) )

Reply:
Good point. Done